spec/lib/project_builder_spec.rb

```diff
@@ -5,20 +5,40 @@
     allow(LookupContentIdWorker).to receive(:perform_async)
   end
 
+  let(:project_name) { 'project_name' }
+  let(:taxonomy_branch_content_id) { SecureRandom.uuid }
+  let(:content_item_attributes_enum) { [] }
+  let(:bulk_tagging_enabled) { false }
+
+  def subject(
```

Comment:
Would it make sense to name this `create_project` instead of `subject`?

Then:

```rb
expect { subject }.to change { Project.count }.by(1)
```

Becomes self-explanatory as:

```rb
expect { create_project }.to change { Project.count }.by(1)
```